pyproject.toml

```diff
@@ -97,6 +97,7 @@ langchain = "^0.1.4"
 requests = "^2.31.0"
 openai = ">=1.1.1"
 chromadb = "^0.5.0"
+lancedb = "^0.6.2"
```

Comment:
As we are making the package lighter #1391 please remove library from here and prompt the user to install `lancdb` when required.

Reply:
sure